blogposts/2021/career-framework.md

```diff
@@ -0,0 +1,316 @@
+
+Individually, engineering managers and ICs cared about career growth but the conversation around growth was unclear and at times uneven. We didn’t have a common language and without a shared understanding of growth, misunderstanding and bias would eventually be inevitable.
+
+The solution? A framework that would align the entire engineering organization and create consistent, fair, and scalalabe conversations about growth. This is the story of how we collaborated to form an engineering career development framework.
+
+If you want to skip to the results, you can [read the handbook page](https://about.sourcegraph.com/handbook/engineering/career-development) or read the summary in this post. But if you want the story, and to see how you can apply our learnings at your company, read on.
+
+## Engineering managers, assemble!
+
+To create the career development framework, our VP of Engineering Nick Snyder formed a working group. We took on the [DACI framework](https://www.atlassian.com/team-playbook/plays/daci), meaning one person would drive the project, one person would approve the project’s results, a select few would contribute to the project, and everyone else would be informed.
+
+The working group consisted of four people:
+
+- [Chris Pine](https://about.sourcegraph.com/handbook/company/team#chris-pine-he-she-they-chris), engineering manager, was the driver.
+- [Nick Snyder](https://about.sourcegraph.com/handbook/company/team#nick-snyder-he-him), VP of Engineering, was the approver.
+- [Felix Becker](https://about.sourcegraph.com/handbook/company/team#felix-becker) and [Patrick Dubroy](https://about.sourcegraph.com/handbook/company/team#patrick-dubroy-he-him), both engineering managers, were contributors.
+
+Everyone else in engineering was to be informed of the project’s progress and results.
+
+The team, sans Nick, met once a week for half an hour each. We did most of the work [asynchronously](https://about.sourcegraph.com/handbook/company/asynchronous-communication) and reported, weekly, to engineering leadership.
+
+In accordance with our [transparency value](https://www.google.com/url?q=https://about.sourcegraph.com/handbook/company/values%23open-and-transparent&sa=D&source=editors&ust=1633036256480000&usg=AOvVaw1CtjufBG3pXsNohoNQ25Wa), we worked in the open. All proposals were in Google Docs that we shared with the entire company. We also asked engineering managers to bring it up in their one-one-ones. If people weren’t comfortable with that directness, we also created an anonymous feedback form.
+
+Even candidates got a peek at the framework:
+
+![Nick, informing the working group via Slack, that he shared our work with a candidate.](IMAGE TK)
```

Comment:
@rebeccadee What's the best way to get an SVG of the image that'll go here? Is that something we can go to Felix for?

Reply:
@nickmyyz Felix gave some directions here: https://github.com/sourcegraph/about/pull/4814#issue-1008153583